src/Tizen.Multimedia.MediaPlayer/Player/MediaStreamSource.cs

```diff
@@ -42,7 +44,39 @@ public static IEnumerable<MediaFormatAudioMimeType> SupportedAudioTypes
         {
             get
             {
-                yield return MediaFormatAudioMimeType.Aac;
+                NativePlayer.Create(out _playerHandle).ThrowIfFailed(null, "Failed to create player");
+                _handle = _playerHandle.DangerousGetHandle();
+                Debug.Assert(_handle != IntPtr.Zero);
+
+                try
+                {
+                    List<MediaFormatAudioMimeType> audioFormats = new List<MediaFormatAudioMimeType>();
+
+                    NativePlayer.SupportedMediaFormatCallback callback = (int format, IntPtr userData) =>
+                    {
+                        if (!Enum.IsDefined(typeof(MediaFormatAudioMimeType), format))
```

Comment:
How about creating `audioFormat` and `videoFormat` at once?
For example,
```csharp
if (Enum.IsDefined(typeof(MediaFormatAudioMimeType), format))
{
      audioFormats.Add((MediaFormatAudioMimeType)format);
}
else
{
      videoFormats.Add((MediaFormatVideoMimeType)format);
}
```

If so, we can reduce pinvoke call.

Reply:
And if you modify like above, you can make this code block(line 53~71) private common method, 
which can be used both properties.

Reply:
actually there are much more formats in native header than c# header. i am worried `else` could get wrong with what i intend to.
if these methods were not divided, it could have covered every types at once..

Reply:
If you worried about `container`, `text`, `unknown` format, you can check it also like the following.
```csharp
if (Enum.IsDefined(typeof(MediaFormatAudioMimeType), format))
{
      audioFormats.Add((MediaFormatAudioMimeType)format);
}
else if (Enum.IsDefined(typeof(MediaFormatVideoMimeType), format))
{
      videoFormats.Add((MediaFormatVideoMimeType)format);
}
else
{
      Log.Debug(PlayerLog.Tag, "It's not audio/video format. skipped : " + format.ToString());
}
```

Reply:
then i'll combine it to one method